go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/go-pg/pg/v9
+module github.com/tpoxa/pg/v9
```

Comment:
Please don't change the package when submitting a pull request. Go mod should allow you to do any changes you need without needing to do this.